### ISSUE TITLE
Certified for master branch docs needs is not valid

### ISSUE THREAD

**Issue by camilamacedo86** [kubernetes-sigs/kubebuilder#3329]
### What broke? What's expected?

The docs published from the master branch: https://master.book.kubebuilder.io/plugins/creating-plugins.html is complaining about an invalid certification. More info: https://github.com/operator-framework/operator-sdk/issues/6387

We need to check how to renew the net fly certified used. The same issue does not occur for the production docs. 

You can learn more about how the docs are configured by looking at the CONTRIBUTING guide: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#how-to-contribute-to-docs

### Reproducing this issue

Access the master branch docs;

<img width="1058" alt="Screenshot 2023-04-12 at 10 06 18" src="https://user-images.githubusercontent.com/7708031/231409456-8a33e3fd-06e3-45a0-a6bf-0c45bd971e46.png">

### KubeBuilder (CLI) Version

master

**Comment by pratikmota** [comment]
I have done some analysis on this issue.
I am getting a certificate SSL warning for book-v1, book-v2 and master all 3 subdomain websites.
Error message is different for v1/v2 and master.
**Master:** The certificate is only valid for the following names: ***.netlify.app, netlify.app**
**book-v1/v2 :** The certificate is only valid for the following names: ***.kubebuilder.io, kubebuilder.io**

So, For master subdomain. It is expecting *.netlify.app domain only.
My assumption( not sure):
  Maybe somehow certificate is changed from  **Custom certificates** to **Netlify-managed certificates**

Here is SSL warning I am getting while opening in Firefox:

1) https://master.book.kubebuilder.io/
Websites prove their identity via certificates.
Firefox does not trust this site because it uses a certificate that is not valid for master.book.kubebuilder.io. 
**The certificate is only valid for the following names:** ***.netlify.app, netlify.app**
Error code: SSL_ERROR_BAD_CERT_DOMAIN
Unable to communicate securely with peer: requested domain name does not match the server’s certificate.

2) https://book-v1.book.kubebuilder.io/
Websites prove their identity via certificates. 
Firefox does not trust this site because it uses a certificate that is not valid for book-v1.book.kubebuilder.io. 
**The certificate is only valid for the following names:** ***.kubebuilder.io, kubebuilder.io**
Error code: SSL_ERROR_BAD_CERT_DOMAIN
Unable to communicate securely with peer: requested domain name does not match the server’s certificate.

3) https://book-v2.book.kubebuilder.io/
Websites prove their identity via certificates. 
Firefox does not trust this site because it uses a certificate that is not valid for book-v2.book.kubebuilder.io. 
**The certificate is only valid for the following names:** ***.kubebuilder.io, kubebuilder.io**
Error code: SSL_ERROR_BAD_CERT_DOMAIN
Unable to communicate securely with peer: requested domain name does not match the server’s certificate.

**Few URLs from internet may be helpful for us:**
1) https://answers.netlify.com/t/certificate-is-not-valid/56500/2
Summary of article:
	The solution was to renew the certificate after changing the domain information (which at this time was not specified in the docs)
	Options found under: Site settings > Domain management > HTTPS > Renew Certificate
	
2) https://docs.netlify.com/domains-https/https-ssl/
Two type of SSL certificates:
	1) Netlify-managed certificates
	   Contains like: https://sub-domain.netlify.app
	2) Custom certificates
             without netlify. custom domain

**Comment by varshaprasad96** [comment]
The conversation regarding this is here: https://kubernetes.slack.com/archives/C1TU9EB9S/p1681331310546959

Tl;dr: The site for the master branch seems to work (https://master.book.kubebuilder.io/) but the `book-v1` doesn't (https://book-v1.book.kubebuilder.io/). There are no specific options which can be changed either, since to the admin it shows that the certificates are renewed. We may have to open a ticket with netlify and figure out what's happening.

**Comment by pratikmota** [comment]
Looks like the given issue is resolved ??
I am not getting any SSL warning for all 3 sub-domains.

1. https://master.book.kubebuilder.io/

![image](https://user-images.githubusercontent.com/5825319/231742673-8c89a818-958b-4ee8-be8c-bf33762c53bc.png)

3. https://book-v1.book.kubebuilder.io/

![image](https://user-images.githubusercontent.com/5825319/231742848-91b0033d-7d32-4445-b4c1-232fcd70a19d.png)

5. https://book-v2.book.kubebuilder.io/

![image](https://user-images.githubusercontent.com/5825319/231743042-df6053da-0010-4428-a2c9-fc6e722c2455.png)

**Comment by varshaprasad96** [comment]
Great! No idea how it fixed on its own, but I'm glad it did :)
